tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -398,7 +407,7 @@ def test_filter(self, xp, scp):
             'filter': ['maximum_filter', 'minimum_filter'],
             'origin': [0, (0, 1, 1, 0)],
             'ksize': [(3, 3, 4, 5)],
-            'mode': [('reflect', 'nearest', 'mirror', 'wrap')],
+            'mode': [('reflect', 'nearest', 'constant', 'wrap')],
```

Comment:
Q: `mirror` is no longer supported by max/min filters?

Reply:
It is still supported, let me add it back. I think I wanted to add `constant` so that the `cval` parameter is relevant to at least one of the cases specified.

Reply:
have added mirror back in